src/Microsoft.ServiceFabric.Services.Remoting/V2/Runtime/IExceptionSerializerSettings.cs

```diff
@@ -0,0 +1,17 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using Microsoft.ServiceFabric.Services.Remoting.FabricTransport.Runtime;
+
+namespace Microsoft.ServiceFabric.Services.Remoting.V2.Runtime
+{
+    interface IExceptionSerializerSettings
+    {
+        int RemotingExceptionDepth { get; }
+        #pragma warning disable 618
```

Comment:
Just a nit, since we're doing this in the release branch and it's fine as is. VS insists on placing compiler directives in the beginning of the line, at least it does for #ifs. Because of this, I think the typical code style is to place directives at the start of the line and keep the code alignment as is. In other words, the ExceptionSerializationTechnique property would be at the same level as the RemotingExceptionDepth. It kind of makes sense logically as well, since they are both properties of this interface.